ui/index.html

```diff
@@ -11,7 +12,7 @@
       </div>
       <div id="auditNameContainer" class="container" style="display:none">
          Enter a name for this audit:<br />
-         <input type="text" id="auditNameBox" />
+         <input type="text" id="auditNameBox" maxLength="15" onkeypress="limitKeypress(event,this.value,maxLength)" />
```

Comment:
I think the `maxLengths` should be longer here to accommodate names like `"bristol_audit_1_district_3_test"` or similar

Reply:
Switched the maxLength to 35